R/save_data.R

```diff
@@ -0,0 +1,58 @@
+#' Save extracted data
+#'
+#' Saves any extracted data as separate sheet of an xlsx file.
+#'
+#' @param import The import object to save the extracted data for.
+#' @param file Optional file to save the data to, defaults to creating file in
+#' current working directory called extracted_data.xlsx
+#'
+#' @export
+#'
+#' @examples
+#' import <- new_import(system.file("examples/person_information", package = "dettl"), "test")
+#' run_import(import, "extract")
+#' t <- tempfile()
+#' save_extracted_data(import, t)
+save_extracted_data <- function(import, file = "extracted_data.xlsx") {
```

Comment:
I am not sure about the file argument here.  The name is pretty arbitrary and I wonder if it would be better to either leave it off or use a temporary file if used (with same thought about the transformed version of this below)

Reply:
By this you think remove the default value for the arg or remove the arg altogether?

Reply:
remove the default value for the argument, keep the argument itself